app/assets/stylesheets/Styling Notes

```diff
@@ -0,0 +1,4 @@
+Color Scheme
+
+Black, gold and maybe dark red or white.
+
```

Comment:
usually this can be documented in a scss file where you can store the hex code for the specific colors into sass variables